examples/lua_rockstar.py

```diff
@@ -0,0 +1,5 @@
+from rockstar import RockStar
+
+lua_code = "print 'hello world!'"
+rock_it_bro = RockStar(days=400, file_name='ohai.lua', code=lua_code)
```

Comment:
can change the file name to `helloworld.lua`? 

And also squash the both commits into one?